sar_pipeline/aws/cli.py

```diff
@@ -83,6 +103,14 @@
     type=click.Path(dir_okay=False, path_type=Path),
     help="Path to where the RTC/opera config wil be saved",
 )
+@click.option(
+    "--skip-existing-products",
+    required=False,
+    is_flag=True,
+    default=False,
+    help="Skip the creating burst products if it already exist in the desired s3 bucket path. "
```

Comment:
Skip creating the burst products if they already exist in the desired S3 bucket path.